src/Ui/Menu/AdminMenuListener.php

```diff
@@ -13,7 +13,7 @@ public function __invoke(MenuBuilderEvent $event): void
     {
         $menu = $event->getMenu();
 
-        /** @var ItemInterface $salesMenu */
+        /** @var ItemInterface|ItemInterface[] $salesMenu */
```

Comment:
Do we really need to add this? What was the error behind it?

Reply:
I exclude it. But ItemInterface and Session have IteratorAggregate interface and neon seeing it as iterable object 
![Screenshot 2020-02-12 at 11 38 28](https://user-images.githubusercontent.com/39232096/74327750-00f4eb80-4d8d-11ea-8225-5b923da99256.png)
![Screenshot 2020-02-12 at 11 39 46](https://user-images.githubusercontent.com/39232096/74327753-02beaf00-4d8d-11ea-860d-c66d88633323.png)